openapi/nexus.json

```diff
@@ -17885,6 +17885,47 @@
       },
       "ExternalIp": {
         "oneOf": [
+          {
+            "description": "A source NAT IP address.\n\nSNAT addresses are ephemeral addresses used only for outbound connectivity.",
+            "type": "object",
+            "properties": {
+              "first_port": {
+                "description": "The first usable port within the IP address.",
+                "type": "integer",
+                "format": "uint16",
+                "minimum": 0
+              },
+              "ip": {
+                "description": "The IP address.",
+                "type": "string",
+                "format": "ip"
+              },
+              "ip_pool_id": {
+                "description": "ID of the IP Pool from which the address is taken.",
+                "type": "string",
+                "format": "uuid"
+              },
+              "kind": {
+                "type": "string",
+                "enum": [
+                  "s_nat"
```

Comment:
Not a big deal, but would be slightly nicer if this is `snat`. To get that, I think all you need to do is change `SNat` to `Snat`.

Reply:
I added a `serde(rename = "snat")` directive to accomplish the same thing with no changes to the source. See 3ddf0411c.

Reply:
Nice. That is better.